builddefs/build_keyboard.mk

```diff
@@ -176,6 +176,24 @@ ifeq ($(strip $(CONVERT_TO_PROTON_C)), yes)
     include platforms/chibios/boards/QMK_PROTON_C/convert_to_proton_c.mk
 endif
 
+ifeq ($(strip $(STMC)), yes)
+  CONVERT_TO_STEMCELL=yes
+endif
```

Comment:
`CTPC` was shorthand that I'm not particularly a fan of -- I'm personally not sure I want to set the precedent for allowing shorthand of `STMC = yes` given that it doesn't convey a lot of context. We have the KB2040 and ProMicro RP2040 coming -- `CKB2040 = yes`, `CPM2040 = yes`? I can see it getting a bit... muddy.

Not currently requesting that you change it, but I'd like some weigh-in from other collaborators given the direction.

Reply:
Honestly, that was a bit of my concern too.

Reply:
> Not currently requesting that you change it,

Its enough to block merge in my book, till a longer term solution is in place.